_samples/json/flatten_json.sql

```diff
@@ -0,0 +1,29 @@
+--snowflake functionality to demo how SQL can be used to parse json in a single column 
+
+with json_example as (
+select * from {{ref('json_example')}}
```

Comment:
praise/suggestion: I love this concept, but I  think all examples should be able to run. can you add a json_example model into the models/examples directory?

additionally, you might want to rename this (and the json example) to specify it's `snowflake` specific since the syntax would probably be different for other data platforms

Reply:
if you want help creating an example json source we can build upon lmk :)